examples/README.md

```diff
@@ -1,3 +1,9 @@
+You can run any example by executing
+```
+cargo run --example NAME_OF_THE_EXAMPLE
```

Comment:
Would you mind adding the `--release` flag here? E.g.
```
cargo run --release --example NAME_OF_THE_EXAMPLE
```
This way we can improve the chance that the example performs well on their system and leaves a good first impression :)

Reply:
Done